Snowflake.Data.Tests/IntegrationTests/MaxLobSizeIT.cs

```diff
@@ -290,7 +289,6 @@ public void TestPutGetCommand(int lobSize)
 
         static IEnumerable<int> LobSizeTestCases = new[]
         {
-            SmallSize,
```

Comment:
why did you remove this test case? Isn't it valid any more?

Reply:
No, it's not valid since this test should test different max lob sizes. Also it makes the test flaky because of the behaviour of Snowflake (for small size) when max lob size is increased.